testsuite/features/support/constants.rb

```diff
@@ -948,13 +936,23 @@
       sle-product-suse-manager-proxy-4.3-updates-x86_64
       sle-module-suse-manager-proxy-4.3-pool-x86_64
       sle-module-suse-manager-proxy-4.3-updates-x86_64
+      sle-module-basesystem15-sp4-pool-x86_64-proxy-4.3
+      sle-module-basesystem15-sp4-updates-x86_64-proxy-4.3
+      sle-module-containers15-sp4-pool-x86_64-proxy-4.3
+      sle-module-containers15-sp4-updates-x86_64-proxy-4.3
     ],
   'suma-retail-branch-server-43' =>
     %w[
       sle-product-suse-manager-retail-branch-server-4.3-pool-x86_64
       sle-product-suse-manager-retail-branch-server-4.3-updates-x86_64
       sle-module-suse-manager-retail-branch-server-4.3-pool-x86_64
       sle-module-suse-manager-retail-branch-server-4.3-updates-x86_64
+      sle-module-basesystem15-sp4-pool-x86_64-smrbs-4.3
+      le-module-basesystem15-sp4-updates-x86_64-smrbs-4.3
+      sle-module-server-applications15-sp4-pool-x86_64-smrbs-4.3
+      sle-module-server-applications15-sp4-updates-x86_64-smrbs-4.3
+      sle-module-suse-manager-proxy-4.3-pool-x86_64-smrbs
+      sle-module-suse-manager-proxy-4.3-updates-x86_64-smrbs
```

Comment:
I wonder why we missed those in the first place...

Reply:
Because it's quite hard to make this map.